assets/src/edit-story/components/checklist/karma/checklist.karma.js

```diff
@@ -346,3 +347,248 @@ describe('Checklist integration', () => {
     expect(fixture.editor.checklist.priorityPanel).toBeDefined();
   });
 });
+
+[true, false].forEach((hasUploadMediaAction) => {
+  describe('Checklist integration - hasUploadMediaAction=false', () => {
```

Comment:
Did you intend to make the description dynamic here?
```suggestion
[true, false].forEach((hasUploadMediaAction) => {
  describe(`Checklist integration - hasUploadMediaAction=${hasUploadMediaAction}`', () => {
```

To be fair, I don't find the `forEach` here, the `hasUploadMediaAction` conditional checks in the code and the `it(`should${hasUploadMediaAction ? '' : ' not'}....` very readable.

Can't we just write two separate tests?

Reply:
Yeah no problem, it's a little hard to read. I can write two separate tests